patterns/1-initial/extensions-for-sustainable-growth.md

```diff
@@ -1,23 +1,39 @@
 ## Title
 
-Extensions to Manage Contributions at Scale
+Extensions for Sustainable Growth
 
 ## Patlet
 
-An InnerSource project is receiving too many contributions, making maintenance difficult and resulting in a large code review backlog or premature rejection of new feature contributions. By offering an extension mechanism outside of the core project, the maintainers enable scaling of project capabilities with minimal cost and maintenance overhead.
+An InnerSource project is receiving too many contributions, making maintenance difficult. By offering an extension mechanism outside of the core project, the maintainers enable scaling of project capabilities with minimal cost and maintenance overhead.
 
 ## Problem
 
-As the number of new feature contributions to a mature InnerSource repository rapidly increases, it adds more burden on code reviews and maintenance. How do we allow for faster release of new features, encouraging innovation and experimentation; while also keeping the repository well maintained?
+As the number of contributions to a mature InnerSource repository rapidly increases, it adds more burden on code reviews and maintenance. This results in a large code review backlog or premature rejection of new feature contributions. How do we allow for faster release of new features, encouraging innovation and experimentation; while also keeping the repository well maintained?
+
+## Story
+
+There is a strategic project that aims to collect the best innovations within a domain space to one common stack, allowing reuse of a common infrastructure and providing a standard user experience. Through InnerSource, various teams in the organization that work within the domain space get an opportunity to collaborate and contribute their innovations to the common codebase. However, a large number of contributions in parallel from several developers is making maintenance of the codebase difficult. This is adding a huge burden on the project maintainers who assume ownership over the code quality standards and enable the community through various forms of communication.
+
+Project maintainers are burnt out due to:
```

Comment:
Do you literally mean "burnout" here, or could we use a different phrase?

Just wondering if burnout might be too strong? 
However if that is how it was for the group in your story, by all means, keep it as is.

Reply:
I think burnout would be an appropriate term here that would be caused because of reasons mentioned. I could also remove this paragraph , since the matter is described above `However, a large number of contributions in parallel from several developers is making maintenance of the codebase difficult. This is adding a huge burden on the project maintainers who assume ownership over the code quality standards and enable the community through various forms of communication.`

and essentially we would like extensions pattern to be applied before maintainers face burnout , when maintenance starts to become difficult for a project

Reply:
Feel free to leave the burnout term in there, if this is the extent that you observed in your story.

We could also change it to this. Might make it a bit less strong?
> Project maintainers are at risk of burnout due to:

Reply:
Good idea, updated